src/helpers/filterTodo.ts

```diff
@@ -0,0 +1,10 @@
+import { FilterBy } from '../types/Filter';
+import { Todo } from '../types/Todo';
+
+export const filterTodo = (todos: Todo[], filterBy: FilterBy) => {
+  switch (filterBy) {
+    case FilterBy.Active: return todos.filter(todo => !todo.completed);
+    case FilterBy.Completed: return todos.filter(todo => todo.completed);
+    default: return todos;
```

Comment:
```suggestion
    case FilterBy.Active:
      return todos.filter(todo => !todo.completed);
    case FilterBy.Completed:
      return todos.filter(todo => todo.completed);
    default:
      return todos;
```
Pay attention to formatting